src/components/application_manager/test/commands/mobile/add_command_request_test.cc

```diff
@@ -160,7 +144,7 @@ TEST_F(AddCommandRequestTest, GetRunMethods_SUCCESS) {
 }
 
 TEST_F(AddCommandRequestTest, OnEvent_UI_SUCCESS) {
-  MessageSharedPtr msg_ui = create_fullParams_ui_so();
+  MessageSharedPtr msg_ui = CreateFullParamsUISO();
   using namespace application_manager::commands;
```

Comment:
@wolfylambova22, you already define using for `AddCommandRequest`. This line is redundant.